src/components/learner-credit-management/BudgetDetailApprovedRequestTable.jsx

```diff
@@ -0,0 +1,158 @@
+    ? tableData.requestStatusCounts
+      .filter(
+        ({ requestStatus }) => !!getRequestStatusDisplayName(requestStatus),
+      )
+      .map(({ requestStatus, count }) => ({
+        name: getRequestStatusDisplayName(requestStatus),
+        number: count,
+        value: requestStatus,
+      }))
+    : [];
+
+  const approvedRequestsTableData = (() => ({
+    tableActions: [],
+    additionalColumns: [],
+  }))();
+
+  return (
+    <DataTable
+      isSortable
+      manualSortBy
+      isPaginated
+      manualPagination
+      isFilterable
+      manualFilters
+      isLoading={isLoading}
+      defaultColumnValues={{ Filter: TableTextFilter }}
+      manualSelectColumn={selectColumn}
+      FilterStatusComponent={FilterStatus}
+      SelectionStatusComponent={DataTable.ControlledSelectionStatus}
+      columns={[
+        {
+          Header: intl.formatMessage({
+            id: 'lcm.budget.detail.page.approved.requests.table.columns.request.details',
+            defaultMessage: 'Request details',
+            description:
+              'Column header for the request details column in the approved requests table',
+          }),
+          accessor: 'requestDetails',
+          Cell: RequestDetailsTableCell,
+          disableSortBy: true,
+        },
+        {
+          Header: intl.formatMessage({
+            id: 'lcm.budget.detail.page.approved.requests.table.columns.amount',
+            defaultMessage: 'Amount',
+            description:
+              'Column header for the amount column in the approved requests table',
+          }),
+          accessor: 'amount',
+          Cell: RequestAmountTableCell,
+          disableFilters: true,
+        },
+        {
+          Header: intl.formatMessage({
+            id: 'lcm.budget.detail.page.approved.requests.table.columns.status',
+            defaultMessage: 'Status',
+            description:
+              'Column header for the status column in the approved requests table',
+          }),
+          accessor: 'requestStatus',
+          Cell: RequestStatusTableCell,
+          Filter: CheckboxFilter,
+          filter: 'includesValue',
+          filterChoices: statusFilterChoices,
+        },
+        {
+          Header: intl.formatMessage({
+            id: 'lcm.budget.detail.page.approved.requests.table.columns.recent.action',
+            defaultMessage: 'Recent action',
+            description:
+              'Column header for the recent action column in the approved requests table',
+          }),
+          accessor: 'recentAction',
+          Cell: RequestRecentActionTableCell,
+          disableFilters: true,
+        },
+      ]}
+      additionalColumns={approvedRequestsTableData.additionalColumns}
+      tableActions={approvedRequestsTableData.tableActions}
+      initialTableOptions={{
+        getRowId: (row) => row?.uuid?.toString(),
+      }}
+      initialState={{
+        pageSize: PAGE_SIZE,
```

Comment:
As per figma, there should be three records per page, but you are using it PAGE_SIZE=25 here.